python/cuml/common/base.pyx

```diff
@@ -163,21 +159,13 @@ class Base:
         del base  # optional!
     """
 
-    def __init__(self, handle=None, verbose=False, output_type=None,
-                 verbosity=logger.LEVEL_INFO):
+    def __init__(self, handle=None, verbosity=logger.LEVEL_INFO,
+                 output_type=None):
```

Comment:
After all our issues with imports the PR looks great, but I have one question/concern. Should we keep the `verbose` parameter in the python functions deprecated to avoid breaking existing code for 0.14? cc @JohnZed for his input on this as well.

Reply:
I didn't see a lot of folks using this flag. So, I just directly removed it. However, I'm ok with removing it in stages too.

Reply:
I see two uses in current notebooks (kmeans_mnmg and benchmark) and one in the now-gone simple umap notebook, so it is around. Unfortunately, I think it would be nicer to deprecate for one release, since it affects so many algorithms. We can still remove from docstrings. I think uses of verbose all roll up to setting the parameter in Base so we can just add a warning there, right?

Reply:
Not sure if it is as simple as that. We can certainly add a deprecation warning for `verbose` flag in `Base` class. However, for it to work correctly, we'll have to add such a deprecation warning for all child classes where `verbose` is being used too, right!?

Reply:
no, as long as the verbose parameter is  in the constructor of the child classes and is passed to the super  constructor of the classes and the base class has the warning, then that should be enough.

Reply:
I see. That's done now. Can you re-review?